packages/css-data/src/schema.ts

```diff
@@ -17,13 +17,22 @@ type CustomProperty = `--${string}`;
 
 export type AppliesTo = Properties[StyleProperty]["appliesTo"];
 
-const Unit = z.union([z.enum(units), z.literal("number")]);
+export type UnitGroup = keyof typeof units;
 
-export type Unit = z.infer<typeof Unit>;
+type UnitEnum = typeof units[UnitGroup][number];
+
+const zUnit = z.union([
```

Comment:
convention

Reply:
Undefined behavior :)

Reply:
yes, we discussed it, we need an expert opinion on this @Andarist it turns out typescript/or tooltip in the vscode UI shows any as a type when we use that "shadowing" feature from typescript where schema has the same name as a type.

1. what is the reason it shows us any there, is it a bug or by design? 
2. what can we do about it to avoid confusion?

Reply:
@TrySound until we figured it out please use a convention we use everywhere